src/renderer/components/Experiment/Rag/PickADocumentMenu.tsx

```diff
@@ -1,12 +1,13 @@
 import { Option, Select } from '@mui/joy';
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
 import * as chatAPI from '../../../lib/transformerlab-api-sdk';
 import useSWR from 'swr';
 const fetcher = (url) => fetch(url).then((res) => res.json());
 
 export default function PickADocumentMenu({
   name,
   experimentInfo,
+  docsArray = [],
```

Comment:
we are trying to follow the standard HTML Form naming so we should name this "defaultValue"

See https://mui.com/joy-ui/react-select/